### PR TITLE
docs: add moja global to showcase

### PR DESCRIPTION
**Pull request by sohamsshah**
This PR adds `moja global`'s community website to the Docusaurus Showcase. It is being developed using Docusaurus and we highly appreciate this framework that helped the organization to document work on the website.

[Moja Global](https://github.com/moja-global) is a not for profit, collaborative project that brings together a community of experts to develop open-source software that allows users to accurately and affordably estimate greenhouse gas emissions and removals from the AFOLU sector. The project’s members aim to support the widest possible use of credible emissions estimation software.

On this community website, there is detailed documentation that provides insights about:
* Projects under Moja Global Organization (deliberately described about the usage, stack, implementation etc.)
* Blog Posts from the community. (includes articles, fellowship insights and learnings, cookbooks etc.)
* Community Initiatives (Moja Global has been actively taking initiatives to promote open source via various programs such as Outreachy, LFX, GSOD, GSOC, etc.)
* Community Help Documentation (doubles down upon committees, meetings, mentorship, governance, contributions, guidelines etc.)
* Meetings and notes 
* Case Studies (related to the projects and research)
* Detailed Roadmap of the Projects at the Organization

This project is being actively developed and there are plans to improve and better the UI-UX and performance of the app. Moreover, there are also plans to add multiple language support, add design system integrations and much more. 

I would love if `moja-global` can be added into Showcase section and if the work interests `docusaurus` team, we would love to be listed in `favourites` too! 😉

Again, thanks in advance. Will be active if any changes are needed wrt the PR.

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -808,6 +808,15 @@ const Users: User[] = [
...
   },
+  {
+    title: 'moja global',
+    description: 'Community Website for moja global. moja global is a not for profit, collaborative project that brings together a community of experts to develop open-source software that allows users to accurately and affordably estimate greenhouse gas emissions and removals from the AFOLU sector.',
+    preview: require('./showcase/mojaglobal.png'),
+    website: 'https://community.moja.global/',
+    source: 'https://github.com/moja-global/community-website',
+    tags: ['opensource', 'large']
+
+  },
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:819]:
```suggestion
  {
    title: 'moja global',
    description: 'A non-profit, collaborative open source project to accurately and affordably estimate greenhouse gas emissions and removals from the AFOLU sector',
    preview: require('./showcase/mojaglobal.png'),
    website: 'https://community.moja.global/',
    source: 'https://github.com/moja-global/community-website',
    tags: ['opensource', 'large', 'i18n', 'design'],
  },
```

**Comment by sohamsshah** [comment]
Thanks @Josh-Cena for the review!😊

**Comment by Josh-Cena** [comment]
Yeah, nice site👍 Not sure if it will make into the favorites, waiting on @slorber for that... The deploy preview is also failing, needs a kick on his side

**Comment by sohamsshah** [comment]
> Yeah, nice site👍 Not sure if it will make into the favorites, waiting on @slorber for that... The deploy preview is also failing (seems sharp's binary registry is down), needs a kick on his side

Thanks, awesome!

**Comment by slorber** [comment]
Thanks, that's a nice site

I don't think it's a good fit for favorites though, we keep it for sites that we really want to highlight and have to be selective (ie big org, using fancy features/design...)

**Comment by sohamsshah** [comment]
Thank you so much for adding `moja global` to docusaurus showcase! 🎊🙏

**Pull request merged**